DataFormats/PatCandidates/interface/IsolatedTrack.h

```diff
@@ -0,0 +1,82 @@
+#ifndef __DataFormats_PatCandidates_IsolatedTrack_h__
+#define __DataFormats_PatCandidates_IsolatedTrack_h__
+
+/*
+  \class    pat::IsolatedTrack IsolatedTrack.h "DataFormats/PatCandidates/interface/IsolatedTrack.h"
+  \brief Small class to store key info isolated packed cands    
+   pat::IsolatedTrack stores important info on isolated packed PF candidates,
+   as a convenience to analyses to prevent having to loop over all pfcands.
+  \author   Bennett Marsh
+*/
+
+#include "DataFormats/Candidate/interface/LeafCandidate.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "PhysicsTools/PatUtils/interface/MiniIsolation.h"
+
+namespace pat {
+    class IsolatedTrack;
+    typedef std::vector<IsolatedTrack>  IsolatedTrackCollection; 
+    typedef edm::Ptr<pat::PackedCandidate> PackedCandidatePtr;
+}
+
+
+namespace pat {
+
+    class IsolatedTrack : public reco::LeafCandidate {
+
+      public:
+        typedef MiniIsolation Isolation;
```

Comment:
from the context, it looks like the MiniIsolation better be renamed to PFIsolation and used with this type name everywhere. Better than unnecessary typedefs.